### PR TITLE
Dockerize docker-console

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

**Comment by epistemancer** [comment]
tentatively LGTM, but I'm not familiar with this repo; looks like it's one Platform team owns.  I'll try to get it running locally to check it out further.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM gcr.io/metamarkets-prod-xpn-host/node:8.9.4-alpine
```
Comment (epistemancer) [inline Dockerfile:1]: if it's an easy update, we should try to run this on node 10.16.0, the current LTS version.  I'm planning to get all application services on that node version.

Comment (epistemancer) [reply Dockerfile:1]: furthermore, node:8.9.4-alpine doesn't exist in our container registry, and even if it did, the alpine images don't have python and some other things needed to do the `npm install`.

The current version of the base image I've been building for application services is `gcr.io/metamarkets-prod-xpn-host/node:10.16.1-base-1`.  That's `10.16.1-alpine` with some additional stuff like python installed (can't remember exactly what, have to grab the Dockerfiles off my old laptop).  docker-console appears to build correctly using that image.

**Comment by epistemancer** [comment]
I can get this to build and run locally, but it doesn't work correctly; it keeps throwing errors about service discovery (`can't find druid:prod:master Error: child pool is empty`).  I can manually `disco`, so I'm not sure what's wrong.

**Comment by himadrisingh** [comment]
closing for unified console on router.

**Pull request closed without merging**